lib/cloud_controller/metrics/prometheus_updater.rb

```diff
@@ -4,11 +4,30 @@ module VCAP::CloudController::Metrics
   class PrometheusUpdater
     def initialize(registry=Prometheus::Client.registry)
       @registry = registry
+
+      # Register all metrics, to initialize them for discoverability
```

Comment:
Prometheus best practice is to [initialise all time series to prevent missing metrics](https://prometheus.io/docs/practices/instrumentation/#avoid-missing-metrics).

Reply:
So far I only initialised the metrics, which are not collected by the PeriodicUpdater, because those will be initialised on startup. I don't necessarily like how this is solved currently and might give it some more thoughts.

Reply:
Now all metrics are initialised on creation of the PrometheusUpdater object.